apps/web/src/routes/AuthLayoutRouter/AuthLayoutRouter.js

```diff
@@ -420,11 +420,16 @@ export default function AuthLayoutRouter (props) {
               <Route path='groups/:groupSlug/*' element={homeRoute} />
               <Route path='post/:postId/*' element={<PostDetail />} />
               {/* **** My Routes **** */}
-              <Route path='my/:view/*' element={<Stream context='my' />} />
+              <Route path='my/posts/*' element={<Stream context='my' />} />
```

Comment:
we may run into an issue losing the :view parameter from the URL params? not sure. and there may be other ways to handle that

Reply:
I passed them in as props instead, in this instance